addons/sale_project/models/sale_order_line.py

```diff
@@ -19,58 +19,15 @@ class SaleOrderLine(models.Model):
         'project.task', 'Generated Task',
         index=True, copy=False)
     # used to know if generate a task and/or a project, depending on the product settings
-    is_service = fields.Boolean("Is a Service", compute='_compute_is_service', store=True, compute_sudo=True)
```

Comment:
Probably needs a migration script to avoid a field recomputation on all the SO lines.